fakeredis/_fakesocket.py

```diff
@@ -14,6 +29,12 @@
 from .commands_mixins.string_mixin import StringCommandsMixin
 from .commands_mixins.transactions_mixin import TransactionsCommandsMixin
 
+if TYPE_CHECKING:
```

Comment:
Please just leave the inheritance from `JSONCommandsMixin` here, avoid the type-checking for now.

Reply:
Applied